### PR TITLE
Update deps

### LINKED ISSUE #21: Support for Rider 2021.1
This plugin does not support Rider > 2020.3.2
Is there any reason for that or can you upgrade the dependency to 2021.1

### PR DESCRIPTION
**Pull request by erri120**
Resolves #21

### CODE REVIEW

File: src/ReSharper.Structured.Logging/ReSharper.Structured.Logging.Rider.csproj [hunk 1]
```diff
@@ -10,7 +10,10 @@
     <RootNamespace>ReSharper.Structured.Logging</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="JetBrains.Rider.SDK" Version="2020.3.2" />
+    <PackageReference Include="JetBrains.Rider.SDK" Version="2021.1.0-eap06">
```
Comment (StormPooper) [inline src/ReSharper.Structured.Logging/ReSharper.Structured.Logging.Rider.csproj:13]: The [non-EAP](https://www.nuget.org/packages/JetBrains.Rider.SDK/2021.1.0) version is now available.
```suggestion
    <PackageReference Include="JetBrains.Rider.SDK" Version="2021.1.0">
```

File: src/ReSharper.Structured.Logging/ReSharper.Structured.Logging.csproj [hunk 2]
```diff
@@ -5,7 +5,10 @@
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="JetBrains.ReSharper.SDK" Version="2020.3.2" />
+    <PackageReference Include="JetBrains.ReSharper.SDK" Version="2021.1.0-eap06">
```
Comment (StormPooper) [inline src/ReSharper.Structured.Logging/ReSharper.Structured.Logging.csproj:8]:
```suggestion
    <PackageReference Include="JetBrains.ReSharper.SDK" Version="2021.1.0">
```

File: test/src/ReSharper.Structured.Logging.Rider.Tests.csproj [hunk 3]
```diff
@@ -4,7 +4,10 @@
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="JetBrains.Rider.SDK.Tests" Version="2020.3.2" />
+    <PackageReference Include="JetBrains.Rider.SDK.Tests" Version="2021.1.0-eap06">
```
Comment (StormPooper) [inline test/src/ReSharper.Structured.Logging.Rider.Tests.csproj:7]:
```suggestion
    <PackageReference Include="JetBrains.Rider.SDK.Tests" Version="2021.1.0">
```

File: test/src/ReSharper.Structured.Logging.Tests.csproj [hunk 4]
```diff
@@ -4,7 +4,10 @@
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="JetBrains.ReSharper.SDK.Tests" Version="2020.3.2" />
+    <PackageReference Include="JetBrains.ReSharper.SDK.Tests" Version="2021.1.0-eap06">
```
Comment (StormPooper) [inline test/src/ReSharper.Structured.Logging.Tests.csproj:7]:
```suggestion
    <PackageReference Include="JetBrains.ReSharper.SDK.Tests" Version="2021.1.0">
```

**Comment by olsh** [comment]
Thank you for the contribution 👍🏻 
This was fixed in 6a4bc53d860416d1bec5edb4d3bb89b859e969d7
There was a tricky part with tests.

**Pull request closed without merging**